geth/transactions/transactor.go

```diff
@@ -197,8 +198,21 @@ func (t *Transactor) validateAndPropagate(selectedAccount *account.SelectedExtKe
 	}
 	ctx, cancel = context.WithTimeout(context.Background(), t.rpcCallTimeout)
 	defer cancel()
+
 	if err := t.sender.SendTransaction(ctx, signedTx); err != nil {
 		return hash, err
 	}
 	return signedTx.Hash(), nil
 }
+
+func (t *Transactor) overrideSignTxArgs(signArgs *sign.TxArgs, args *SendTxArgs) {
```

Comment:
I need to read the body of this method to figure out what overrides what. Maybe something like `mergeSignTxArgsOntoSendTxArgs()`?

Reply:
Oh, and do we need `sign.TxArgs` while we have `SendTxArgs`? Or that will result in too tight coupling of packages?

Reply:
yes, it will result in an import cycle as its used on [sign/pending_requests.go](https://github.com/status-im/status-go/pull/1000/files#diff-5d395fef9ef558fe57b242d7608db40fL11)